src/emulate.c

```diff
@@ -346,13 +350,7 @@ static block_t *block_find(const block_map_t *map, const uint32_t addr)
 
 FORCE_INLINE bool insn_is_misaligned(uint32_t pc)
```

Comment:
I wonder if `insn_is_misaligned` is really used via macro expansion or not. Can you check?

Reply:
I inserted a ```fprintf``` statement inside ```insn_is_misaligned()``` to verify if the function is being executed. It should not run during ```make ENABLE_EXT_C=1 check``` but should run during ```make ENABLE_EXT_C=0 check```. My experimental results align with my expectations.

Reply:
Then, `insn_is_misaligned` should be conditionally built, depending on the configurations.